src/views/Home.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+import QuestionCard from "../components/QuestionCard";
+import Hero from "../components/Hero";
+
+export default function Home(props) {
+  const list_questionID = ['1', '2']
```

Comment:
Your mixing snake case and camel case. JavaScript convention is to use camel case, let's use it everywhere.

Reply:
fix in commit 5e56fde